### ISSUE TITLE
Unable to create DB for a particular folder under repo (folder name have space in it e.g. 'test folder')

### ISSUE THREAD

**Issue by GoogleCodeExporter** [waterxi/svnplot#37]
```
What steps will reproduce the problem?
1. Repository contains folder named 'test folder' (please consider space in
folder name)
2. Execute command - svnlog2sqlite.py -l
http://servername/reponame/repo/trunk/test folder f:\sqliteDB\test_db

Error - Error Unable to find repository location for
"http://servername/reponame/trunk/test folder" in revision 1

What is the expected output? What do you see instead?
Expected output -  DB should be created.
Current output -  Error - (as above)


What version of the product are you using? On what operating system?
1. Product verison - SVNPlot-0.6.0.win32.exe
2. OS - Windows XP


Please provide any additional information below.

Currently, repository has folders viz /trunk/test1 & /trunk/test folder
svnlog2sqlite.py -l is 
working correctly for '/trunk/test1' (folder name without space), but
Error for '/trunk/test folder' (folder name with space)

I have tried 'svnlog2sqlite' cmd with "" also
i.e. svnlog2sqlite.py -l "http://servername/reponame/repo/trunk/test
folder" f:\sqlite\test2_db

In this case, getting error as
ERROR:root:Error URL 'http://servername/reponame/repo/trunk/test f
older' is malformed or the scheme or host or path is missing



```

Original issue reported on code.google.com by `sachinen...@gmail.com` on 19 Apr 2010 at 12:53

**Comment by GoogleCodeExporter** [comment]
```
Try "http://servername/reponame/repo/trunk/test%20folder" as repository URL.
```

Original comment by `nitinbh...@gmail.com` on 4 May 2010 at 7:11

**Comment by GoogleCodeExporter** [comment]
```
Tried
1. http://servername/reponame/repo/trunk/test%20folder/
2. http://servername/reponame/repo/trunk/test%20folder
3. "http://servername/reponame/repo/trunk/test 20folder/"

Still it is giving error as - unable to find repository location for <...given
path...> in revision 1
```

Original comment by `sachinen...@gmail.com` on 4 May 2010 at 8:20

**Comment by GoogleCodeExporter** [comment]
Original comment by `nitinbh...@gmail.com` on 8 Jan 2011 at 5:02
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
fixed in trunk. Will be available 0.7.4
```

Original comment by `nitinbh...@gmail.com` on 8 Jan 2011 at 5:02
- Changed state: **Fixed**
